src/components/Select/Select.types.ts

```diff
@@ -104,6 +104,11 @@ export interface SelectProps
    * The Select input custom class names.
    */
   inputClassNames?: string;
+  /**
+   * The Select input is readonly.
+   * @default false
+   */
+  inputReadOnly?: boolean;
```

Comment:
does this require the word "input"? would love to dial down the verbosity if possible